### DISCUSSION TITLE
DatePicker trigger open pop-up

### DISCUSSION THREAD

**Discussion by dinacuatro** [primefaces/community#2632] [PrimeVue]
I have two DatePicker components in my page, one for start-date and the second for end-date.
I would like to open automatically the second one when a date is selected in the first one.
Is it possible?

**Comment by wnmzzzz** [comment]
Can't you just listen for the blur event on the first picker (or watch the underlying ref) and then use plain javascript to open the second one, ie by simulating a click on it?

**Reply by dinacuatro** [reply]
I tried, but id doesn't work.
